index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio Site</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="//use.fontawesome.com/releases/v5.0.7/css/all.css">
+  
+</head>
+<body>
+  <header class="head">
+    <nav>
+      <a href="#">Portfolio</a>
+      <i class="fa fa-bars"></i>   
+    </nav>
+  </header>
+  <main>
+    <section class="container">
+      <h1>
+      I'm Uchechi<br>
+      Glad to see<br>
+      you!
+      </h1>
+      <p>
+      I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <h3>LETS CONNECT</h3>
+      <ul>
+        <li><img src="images/Tweeter.png"></li>
+        <li><img src="images/Linkedin icon.png"></li>
+        <li><img src="images/m.png"></li>
+        <li><img src="images/Github.png"></li>
+        <li><img src="images/tech.png"></li>
+      </ul>
+    </section>
```

Comment:
Good job changing this to list elements however, kindly change them to lists of likns that is `<li><a><img></a></li>`
This icons should be links to redirect a user your accounts: `github`, `twittter`. 
Its also a requirement to pass this milestone.